subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_connmon.c

```diff
@@ -125,7 +138,63 @@ static void modem_signal_update(struct k_work *work)
 	timestamp_prev = k_uptime_get_32();
 }
 
-int lwm2m_init_connmon(void)
+static void lwm2m_update_connmon_cell(void)
+{
+	k_work_submit(&modem_data_work);
```

Comment:
I wrote a comment about this earlier, but seems that I didn't add the comment by mistake.

`modem_data_work` and `modem_data_update` are very generic names. The updated parameters are related to the registration, but because this function is already named `_cell`, maybe you there could be the word "cell" also in the work and work handler function names. There's also `modem_signal_work` and `modem_signal_update`, so these could be for example `modem_cell_work` and `modem_cell_update`.